libraries/AP_HAL/RCOutput.cpp

```diff
@@ -76,7 +76,7 @@ bool AP_HAL::RCOutput::is_dshot_protocol(const enum output_mode mode)
 /*
  * calculate the prescaler required to achieve the desire bitrate
  */
-uint32_t AP_HAL::RCOutput::calculate_bitrate_prescaler(uint32_t timer_clock, uint32_t target_frequency, bool is_dshot)
+uint32_t AP_HAL::RCOutput::calculate_bitrate_prescaler(uint32_t timer_clock, uint32_t target_frequency, bool at_least_freq)
```

Comment:
the callers pass in an is_dshot. The test suite sometimes passes true and sometimes false for this variable. So the users of this function don't seem to match the test suite or the new function form?